data/src/main/kotlin/com/acon/data/repository/UploadRepositoryImpl.kt

```diff
@@ -0,0 +1,50 @@
+package com.acon.data.repository
+
+import com.acon.data.datasource.remote.UploadRemoteDataSource
+import com.acon.domain.model.upload.DotoriCount
+import com.acon.domain.model.upload.KeyWord
+import com.acon.domain.model.upload.SpotVerification
+import com.acon.domain.model.upload.Suggestions
+import com.acon.domain.repository.UploadRepository
+import javax.inject.Inject
+
+class UploadRepositoryImpl @Inject constructor(
+    private val uploadRemoteDataSource: UploadRemoteDataSource
+) : UploadRepository {
+    override suspend fun getDotoriCount(): Result<DotoriCount> = runCatching {
```

Comment:
```suggestion
    override suspend fun getDotoriCount(): Result<DotoriCount> = runCatchingWith {
```
runCatchingWith로 바꿔주세용

Reply:
아래 다른 부분들도!!